beacon-chain/core/helpers/validators_test.go

```diff
@@ -541,7 +541,13 @@ func TestComputeProposerIndex(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := ComputeProposerIndex(tt.args.validators, tt.args.indices, tt.args.seed)
```

Comment:
maybe you can add a test to test `ComputeProposerIndex` against `ComputeProposerIndexWithValidators` to see that they are always alligned

Reply:
`ComputeProposerIndexWithValidators` is only used for one archival method which is about to be deprecated soon. Not sure if its worth it to write a compatibility test there. If you feel strongly about it, I can add the test in

Reply:
nm sounds fair